modules/nf-core/svtyper/svtyper/main.nf

```diff
@@ -0,0 +1,56 @@
+
+process SVTYPER_SVTYPER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::svtyper=0.7.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/svtyper:0.7.0--py27h24bf2e0_1':
+        'quay.io/biocontainers/svtyper:0.7.0--py27h24bf2e0_1' }"
+
+    input:
+    tuple val(meta), path(bam), path(bam_index), path(vcf)
+    tuple val(meta2), path(fasta), path(fai)
+
+    output:
+    tuple val(meta), path("*.json")        , emit: json
+    tuple val(meta), path("*.gt.vcf")      , emit: gt_vcf
+    tuple val(meta), path("*.relevant.bam"), emit: bam
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def vcf  = vcf ? "--input_vcf ${vcf}" : ""
+
+    """
+    svtyper \\
+        $vcf \\
+        --bam $bam \\
+        --lib_info ${prefix}.json \\
+        --output_vcf ${prefix}.gt.vcf \\
```

Comment:
```suggestion
        --output_vcf ${prefix}.vcf \\
```
Don't use custom prefixes aside from the file extension :) The pipeline developers should add this themselves through the prefix (or not if they don't want to)

Reply:
You can also add a check for file name collisions as shown [here](https://nf-co.re/docs/contributing/modules#naming-conventions) part 6

Reply:
gt.vcf was the way that developers use. Besides, what if input and output names became the same at the end? How will the output be differentiated? That did not make sense to be

Reply:
You can read the link I sent you in my previous message, this explains everything. You can still add the `.gt` yourself through prefix if you want to use it :)